bundle/src/main/java/com/cognifide/cq/cqsm/core/progress/ProgressHelper.java

```diff
@@ -45,4 +48,18 @@ public static String toJson(List<ProgressEntry> entries) {
 	public static String toJson(ProgressEntry entry) {
 		return GSON.toJson(entry);
 	}
+
+	public static boolean calculateSuccess(List<ProgressEntry> entries) {
+
+		boolean success = !Iterables.any(entries, new ProgressHelper.IsErrorPredicate());

```

Comment:
You should return value immediately, without creating success variable.

Reply:
Fixed